src/peft/tuners/lora.py

```diff
@@ -322,10 +344,13 @@ def _create_and_replace(
         parent,
         **optionnal_kwargs,
     ):
+        current_key = optionnal_kwargs["current_key"]
```

Comment:
```suggestion
        current_key = optional_kwargs["current_key"]
```

The typo was already there before but I think since it's a private method, we can fix it there without BC breaking.

Reply:
Done